packages/js-sdk/src/session/process.ts

```diff
@@ -100,20 +100,24 @@ export class Process {
    * Sends data to the process stdin.
    *
    * @param data Data to send
+   * @param opts Call options
+   * @param {timeout} [opts.timeout] Timeout in milliseconds
    */
-  async sendStdin(data: string): Promise<void> {
-    await this.session.call(processService, 'stdin', [this.processID, data])
+  async sendStdin(data: string, opts?: CallOpts): Promise<void> {
+    await this.session.call(processService, 'stdin', [this.processID, data], opts)
   }
 }
+export interface ProcessProps {
```

Comment:
maybe `ProcessOpts` would be more consistent within the codebase, `Props` sounds too "React"y

![Screen 2023-09-04 at 15 52 15](https://github.com/e2b-dev/e2b/assets/697301/be78c2c1-eed0-4e6b-ab64-2db6b4302092)